code/global/RPCs.cs

```diff
@@ -35,6 +35,11 @@ public static void ClientOnPlayerSpawned(TTTPlayer player)
 
             player.SetRole(new NoneRole());
 
+            if (player.Controller is DefaultWalkController playerController)
+            {
+                playerController.UnderwaterBreathSeconds = DefaultWalkController.MAX_UNDERWATER_BREATH_SECONDS;
```

Comment:
This isn't set on serverside

Reply:
I'd like to remove this keeping it as a networked var

Reply:
Could you handle this part? I'm not sure how to go about it.

Reply:
[Net] (https://wiki.facepunch.com/sbox/Networked_Types)